manifests/0000_30_cluster-api_09_admission-policies.yaml

```diff
@@ -152,6 +152,68 @@ data:
             )
           message: "Cannot modify a Cluster API controlled label except to match the Cluster API mirrored machine. This is because status.authoritativeAPI is set to Cluster API."
 ---
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
These need to be in the 'components' key of the above config map, the operator then applies them dynamically! 

These wont be picked up by the test suite like this, either :)